### PR TITLE
Fix local port regex in listen_ports_facts

### PR DESCRIPTION
**Pull request by gaetan-craft**
##### SUMMARY
This PR fix the bug reported in #4091 by fixing the regex used to parse ss local connection entry
##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME
listen_ports_facts

```

### CODE REVIEW

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)? Thanks!

**Comment by gaetan-craft** [comment]
> Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)? Thanks!

Sure, done

File: changelogs/fragments/4092-fix_local_ports_regex_listen_ports_facts.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - listen_ports_facts - local port regex was not handling well ipv6 only binding. Fixes the regex for ss
```
Comment (felixfontein) [inline changelogs/fragments/4092-fix_local_ports_regex_listen_ports_facts.yaml:2]:
```suggestion
  - listen_ports_facts - local port regex was not handling well IPv6 only binding. Fixes the regex for ``ss`` (https://github.com/ansible-collections/community.general/pull/4092).
```

**Review by felixfontein — APPROVED** [review]
Looks formally good to me. Would be glad if someone else could check / test this.

**Comment by felixfontein** [comment]
CC @moonrail since you added `ss` support to this module.

**Comment by felixfontein** [comment]
I did some very basic testing, and it looks correct to me. If nobody complains by tomorrow evening (i.e. in roughly 24 hours), I'll merge.

**Comment by felixfontein** [comment]
@gaetan-craft thanks for fixing this!

**Pull request merged**